avoid confusing cluster name with health

leafnunes opened the pull request:
Clusters with `green` or `yellow` in the cluster name will always trigger certain checks.

leafnunes committed:
avoid confusing cluster name with health

Clusters with `green` or `yellow` in the cluster name will always trigger certain checks.

erik-stephens commented:
Good catch!  Not sure if warrants holding up this PR, but we might want to consider `/_cat/health?h=status` while we're visiting these lines.

leafnunes committed:
Specify the status health parameter

-G is needed to use -d to specify parameters with -d, and implies making a GET request, so -XGET is redundant.

leafnunes commented:
`h=status` is much better that adding a call to cut. Thank you for pointing it out!

erik-stephens reviewed (approved):
Looks good to me.
